src/effects/builtin/builtinbackend.cpp

```diff
@@ -22,6 +22,7 @@
 #include "effects/builtin/phasereffect.h"
 #include "effects/builtin/loudnesscontoureffect.h"
 #include "effects/builtin/metronomeeffect.h"
+#include "effects/builtin/noiseeffect.h"
```

Comment:
Could you rename the file to whitenoiseeffect.h/cpp?

Reply:
Okay